app/helpers/navbar_helper.rb

```diff
@@ -2,4 +2,12 @@ module NavbarHelper
   def active_class(link_path)
     current_page?(link_path) ? "active" : ""
   end
+
+  def settings_active_class(project)
```

Comment:
This is not a good approach (does not scale), if we add a new page to the system will we have to add another check here for this new page? Can't you just use `active_class(settings_path)` or something like that?